app/containers/Sessions/containers/QuestionListGroup/QuestionGroupSettingsModal/index.tsx

```diff
@@ -0,0 +1,398 @@
+import React, { useState } from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import {
+  concat,
+  filter,
+  get,
+  includes,
+  mapKeys,
+  snakeCase,
+  set,
+  compact,
+} from 'lodash';
+
+import { numericValidator } from 'utils/validators';
+import { dayOfWeekAsString } from 'utils/dateUtils';
+import binNoBg from 'assets/svg/bin-no-bg.svg';
+import { DAY_NUMBERS } from 'global/constants';
+import { themeColors } from 'theme';
+
+import { QuestionGroup } from 'models/QuestionGroup';
+
+import Modal from 'components/Modal';
+import H3 from 'components/H3';
+import Checkbox from 'components/Checkbox';
+import Radio from 'components/Radio';
+import Text from 'components/Text';
+import Column from 'components/Column';
+import { LocalizedDatePicker } from 'components/DatePicker';
+import { DateInput } from 'components/Input/DateInput';
+import InequalityChooser from 'components/InequalityChooser';
+import Box from 'components/Box';
+import PlusCircle from 'components/Circle/PlusCircle';
+import HoverableBox from 'components/Box/HoverableBox';
+import Img from 'components/Img';
+import Row from 'components/Row';
+
+import messages from './messages';
+import { Input } from '../../../components/QuestionSettings/Settings/Components/styled';
+import VariableChooser from '../../../../VariableChooser';
+import { getTimeString, parseTime } from './utils';
+
+type QuestionGroupSettingsModalProps = {
+  questionGroup: QuestionGroup;
+  updateQuestionGroup: (
+    changes: object,
```

Comment:
I made a wrong suggestion earlier, this take snake_case into account
```suggestion
        changes: Partial<CamelToSnake<QuestionGroup>>,
```

Reply:
I don't see that change applied, is it causing errors?

Reply:
I see that this change is not applied. What is causing errors?